### PR TITLE
add instructions on forking and pull requests to README

### PR DESCRIPTION
**Pull request by skeskali**
As I mentioned on Slack: 
I have what is probably a really simple suggestion, but one that might’ve been overlooked. ​*I think the editorial readme should spell out that people should fork the repository then submit a pull request*​ for any changes that have been made.

That might seem obvious to people who are used to git/github and used to contributing to OSS projects, but I’ve mostly only used github to manage my own projects, and sometimes I forget about forking/pull requests. :blush:

### CODE REVIEW

**Comment by gr2m** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Here you will find a growing repository of documentation for the Editorial Team.
 
 Take a look at [our contribution guide](CONTRIBUTING.md#joining-the-hoodie-editorial-team). We're looking for people of all sorts, so don't be afraid to get involved. If you're passionate about contributing to a great community, we want you! :yellow_heart:
 
+If you have anything you'd like to contribute, please fork this repository and send a Pull Request for your contribution. Hoodie has [a great article](http://hood.ie/blog/contributing-to-hoodie.html) that explains how to fork and edit files. It's especially useful if you're new to the process.
```
Comment (nickcolley) [inline README.md]: Awesome but in this case is it odd to refer to Hoodie in the third person when above we're not?

Comment (Charlotteis) [inline README.md]: I second that comment :)

Comment (gr2m) [inline README.md]: Yeah, let’s fix that. I'm sorry I reviewed it and it slipped trough. I made a follow up issue here:
https://github.com/hoodiehq/editorial/issues/31

Thanks @nickcolley for catching this!!

Comment (skeskali) [inline README.md]: All fixed!

Comment (nickcolley) [inline README.md]: Sound, looks great but generally it's good if you can let someone else review and merge your work even if it's really trivial, sorry for being such a nit picker, this is great :+1:

Comment (skeskali) [inline README.md]: @nickcolley I worked through it with @Charlotteis as I was actively writing it, and @gr2m reviewed it as I was waiting for my merge permissions to become active.

**Pull request merged**